src/components/Footer.tsx

```diff
@@ -0,0 +1,83 @@
+import {
+  Box,
+  Button,
+  chakra,
+  Container,
+  Stack,
+  Text,
+  useColorModeValue,
+  VisuallyHidden
+} from "@chakra-ui/react";
+import { FaGithub, FaInstagram, FaFacebook } from "react-icons/fa";
+import { ReactNode, useState } from "react";
+
+const SocialButton = ({
+  children,
+  label,
+  href
+}: {
+  children: ReactNode;
+  label: string;
+  href: string;
+}) => {
+  return (
+    <Button
```

Comment:
POTENTIALLY BLOCKING: Is there a reason you didn't use Chakra's `IconButton`? See [here](https://chakra-ui.com/docs/components/icon-button). I think this might fix your icon sizing issue.